doc/usecases/record_elektra/UC_ansible-libelektra_report_conflicts.md

```diff
@@ -0,0 +1,18 @@
+# Use Case: Report merge conflicts in ansible-libelektra
```

Comment:
One merge use case should be enough? This is the Error scenario of the merge use case?

Reply:
Depends on how granular these use cases should be. But yes, we can move them into the same use case.